Profiles - ZIB 2017/zib-FLACCPainScale.xml

```diff
@@ -0,0 +1,353 @@
+          <code value="38217-6" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:Legs.code.coding.display">
+      <extension url="http://hl7.org/fhir/StructureDefinition/elementdefinition-translatable">
+        <valueBoolean value="true" />
+      </extension>
+      <path value="Observation.component.code.coding.display" />
+      <defaultValueString value="Pain severity Legs FLACC" />
+    </element>
+    <element id="Observation.component:Legs.value[x]:valueCodeableConcept">
+      <path value="Observation.component.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <short value="Legs" />
+      <definition value="Observation legs: the extent to which the child is restless (body and legs)." />
+      <alias value="Benen" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <description value="BenenCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.13.2--20171231000000" />
+        </valueSetReference>
+      </binding>
+      <mapping>
+        <identity value="hcim-flaccpainscale-v1.0-2017EN" />
+        <map value="NL-CM:12.13.4" />
+        <comment value="Legs" />
+      </mapping>
+    </element>
+    <element id="Observation.component:Activity">
+      <path value="Observation.component" />
+      <sliceName value="Activity" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:Activity.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="38218-4" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:Activity.code.coding.display">
+      <extension url="http://hl7.org/fhir/StructureDefinition/elementdefinition-translatable">
+        <valueBoolean value="true" />
+      </extension>
+      <path value="Observation.component.code.coding.display" />
+      <defaultValueString value="Pain severity Activity FLACC" />
+    </element>
+    <element id="Observation.component:Activity.value[x]:valueCodeableConcept">
+      <path value="Observation.component.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <short value="Activity" />
+      <definition value="Observation activity: the extent to which the child is restless and has tense muscles." />
+      <alias value="Activiteit" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <description value="ActiviteitCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.13.3--20171231000000" />

```

Comment:
Same as the earlier two comments, please add them. It is just a minor improvement, but a nice one :)